plugin/src/main/java/com/playmonumenta/scriptedquests/managers/SongManager.java

```diff
@@ -55,6 +58,7 @@ public boolean equalsSong(Song song) {
 				&& mSongPath.equals(song.mSongPath);
 		}
 
+		@Subst("minecraft:block.note_block.harp")
```

Comment:
This line was suggested by Adventure for use in static analysis. It has no impact on runtime.